src/components/link/link.stories.mdx

```diff
@@ -95,6 +95,17 @@ The `href` prop provides the ability for the Link to take the user to a differen
   </Story>
 </Preview>
 
+### rel
+
+The `rel` prop provides the ability for the Link to set relationship between the current document 
+and the linked document
+
+<Preview>
+  <Story name="with rel" parameters={{ chromatic: { disable: true } }}>
+    <Link rel="alternate">This is a link</Link>
```

Comment:
I'd go with `<Link target="_blank" rel="noreferrer noopener">...` example as it is most common case.